Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -2616,14 +2620,16 @@ private boolean pathMatch(String templatePath, String path) {
      */
     public static void enableLogging() {
         PrefHelper.LogAlways(GOOGLE_VERSION_TAG);
```

Comment:
Let's add a function for LogAlways in `BranchLogger` so we can totally remove `import android.util.Log;` from PrefHelper.java

Reply:
got it. Removed `LogAlways`, `LogException`, `Warning` and `Debug` functions. Moved usage to `BranchLogger` and removed `import android.util.Log;`